package-overrides/github/devbridge/jQuery-Autocomplete@1.3.0.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": [
+    "dist/",
+    "typings/jquery-autocomplete/",
```

Comment:
Awesome that you included the typings, that is much appreciated. However, there is an issue. The typings directory includes a file named `jquery.autocomplete-tests.ts` which writes directly into the global scope.